src/Controls/src/Core/Button.cs

```diff
@@ -116,12 +116,6 @@ public object CommandParameter
 			set { SetValue(CommandParameterProperty, value); }
 		}
 
-		public Font Font
```

Comment:
this was deleted on all of the other controls (span,label, etc..) so I'm assuming this one was left here by mistake